frontend/src/components/maps/hooks/useSupercluster.ts

```diff
@@ -9,7 +9,7 @@ export interface UseSuperclusterProps<P, C> {
   points: Array<Supercluster.PointFeature<P>>;
   bounds?: BBox;
   zoom: number;
-  options?: Supercluster.Options<P, C>;
+  options?: Supercluster.Options<P, C> & { enableClustering?: boolean };
```

Comment:
I'm not familiar with the syntax.  

My guess below.
- options can be `undefined`
- options is a `Supercluter.Options<P, C>` and must also have a `enableClustering` property that can be `undefined` or `boolean`.

If the above is true, I'm not sure I understand the complexity in the statement?  Wouldn't it be easier to understand by creating an interface with the appropriate inheritance?

Reply:
This is just a shortcut, will switch to using an interface with inheritance.